.github/workflows/translate.yml

```diff
@@ -27,32 +27,22 @@ jobs:
           submodules: recursive
           ref: "master"
 
-      - name: install-deps
-        run: sudo apt install -y qt6-l10n-tools
-
       - name: l10n-remove-old
         run: |
-          rm translations/*.ts
-          git checkout translations/client_en.ts
-
-      - name: l10n-read
-        run: /usr/lib/qt6/bin/lupdate src -no-obsolete -ts translations/client_en.ts
```

Comment:
This line basically generated https://github.com/owncloud/ios-app/tree/master/ownCloud/Resources/en.lproj , which is our translation source. In the next step we then pushed the updated version generated from our sources to transifex.